gamemodes/terrortown/entities/entities/ttt_flame.lua

```diff
@@ -13,54 +13,72 @@ ENT.Model = Model("models/weapons/w_eq_flashbang_thrown.mdl")
 -- @realm shared
 AccessorFunc(ENT, "dmgparent", "DamageParent")
 
----
--- @accessor boolean
--- @realm shared
-AccessorFunc(ENT, "die_explode", "ExplodeOnDeath")
+ENT.firechild = nil
```

Comment:
see, here you're using lowercase as well. I think you should do that everywhere

Reply:
fwiw this one came from upstream

Reply:
ahh, I feel your pain here.

As a rule of thumb: If the var is from upstream, keep it as is. If if is created by you, then keep to our style